packages/sanity-plugin-nacelle-input/src/queries/index.js

```diff
@@ -0,0 +1,69 @@
+export const GET_PRODUCTS = `
+  query allProducts($first: Int, $searchFilter: ProductSearchOptions) {
+    allProducts(filter: { first: $first, searchFilter: $searchFilter }) {
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+      totalCount
+      edges {
+        node {
+          nacelleEntryId
+          content {
+            featuredMedia {
+              thumbnailSrc
+            }
+            handle
+            title
+          }
+          tags
+          productType
+          variants {
+            content {
+              title
+            }
+            sku
+          }
+        }
+      }
+    }
+  }
+`
+
+export const GET_COLLECTIONS = `
+  query allProductCollections(
+    $first: Int
+    $searchFilter: ProductSearchOptions
```

Comment:
I believe this is probably the source of an issue with collection search:

https://user-images.githubusercontent.com/5732000/177665115-954a4f21-2e3a-4085-ba63-d5b9582c4da9.mov



```suggestion
    $searchFilter: ProductCollectionSearchOptions
```